xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/AddressTest.java

```diff
@@ -17,7 +17,7 @@
 import java.util.Optional;
 
 @SuppressWarnings( {"ParameterName", "MethodName", "LocalVariableName"})
-public class XAddressTest {
+public class AddressTest {
```

Comment:
Why the rename here? This should stay `XAddressTest` and should only test the `XAddress` class. Any tests that test `Address` should go in another test class called `AddressTest`